misc/python/materialize/checks/window_functions.py

```diff
@@ -18,6 +18,10 @@ def initialize(self) -> Testdrive:
         return Testdrive(
             dedent(
                 """
+            $ postgres-connect name=mz_system url=postgres://mz_system:materialize@${testdrive.materialize-internal-sql-addr}
+            $ postgres-execute connection=mz_system
+            ALTER SYSTEM SET window_functions_enabled = true
```

Comment:
As discussed in Slack, depending on the rollout procedure, this may cause Nightly upgrade tests to fail, as that statement may be run against a Mz version that does not know about it. If that happens, I will follow up with a PR that fixes the test by introducing some form of conditional execution or conditional skiping of the entire test.